bsp/x86/applications/SConscript

```diff
@@ -4,7 +4,8 @@ from building import *
 
 cwd     = os.path.join(str(Dir('#')), 'applications')
 src	= Glob('*.c')
-CPPPATH = [cwd, str(Dir('#'))]
+drvinc=os.path.abspath('%s/../../../components/drivers/include'%(cwd))
```

Comment:
please not use this way to handle path.

Reply:
but need to include rtdevice.h by DFS, how to

Reply:
#define RT_USING_DEVICE_IPC
in rtconfig.h